Basic/APIDiorama/Assets/UseCaseSamples/SelectionScreen/Scripts/SceneSelectionUI.cs

```diff
@@ -0,0 +1,50 @@
+using DevLocker.Utils;
+using System;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace Unity.Netcode.Samples.APIDiorama
+{
+    [Serializable]
+    internal struct SelectableScene
+    {
+        [SerializeField] internal SceneReference Scene;
+        [SerializeField] internal string DisplayName;
+        [SerializeField] internal Texture2D Image;
+    }
+
+    /// <summary>
+    /// An UI that allows players to pick a scene to load
+    /// </summary>
+    internal class SceneSelectionUI : MonoBehaviour
+    {
+
+        [SerializeField] SelectableScene[] m_Scenes;
+        [SerializeField] GridLayoutGroup m_Container;
+        [SerializeField] SceneSelectionElement m_SceneUIPrefab;
+
+        void OnEnable()
+        {
+            Setup();
+        }
+
+        void Setup()
```

Comment:
Not for this method in particular, but a comment when switching from the SelectionScreen to any one of the individual scenes: it seems like the Directional Light seems to be destroyed from the scene that is being loaded, so there is a visual difference in loading a scene on its own versus going through with the SelectionScreen flow.

Reply:
I'm confused by the change of lights too. 

> it seems like the Directional Light seems to be destroyed from the scene that is being loaded

However if you look at the hierarchy the light object is still there when you load one of the scenes, it is not destroyed. It's still there, but it's working differently.

Another curious fact is that if you start from a scene that has already a light, then go back to the selection screen and then move into another scene from there, there are no lights changes.

Morover, if you make a build that loads the Selection Screen first, and head into any other scene from there, lights work as expected, suggesting that this is an editor-only bug.

A workaround is to add the directional light to the main scene too. I'll mark it as "editor-only" so we don't render useless lights in builds